pkg/runctl/run_test.go

```diff
@@ -32,13 +32,13 @@ const (
     status: Unknown
     type: Succeeded
   podName: build-pod-38aa76
-  startTime: "` + time1 + `"
+  startTime: "2019-05-14T08:24:08Z"
```

Comment:
Put this into a constant, too. Also name the constants according to their meaning.